### ISSUE TITLE
Alternative client

### ISSUE THREAD

**Issue by almassapargali** [davidstump/SwiftPhoenixClient#29]
Hello, wanted to let you know that I've built alternative client on Swift, while I've used many places of this project for ideas, it was total overhaul, so I though making it separate project would be better than PR.

I wouldn't write here about that, but looking at your README looks like you don't have much time for maintenance, and I thought it would be a good idea to let developers know about alternatives, and I don't have any intentions to insult you here.

Basically it supports iOS 8 (#28), Carthage (#10), token authorisation (#9). Otherwise it's almost the same, except that it allows you to pass callbacks for messages to which server `:reply`es (the feature I've needed, and the reason I've built it).

Link to it: https://github.com/almassapargali/PhoenixWebSocket
Thanks, anyway!

**Comment by davidstump** [comment]
Thanks for the details and heads up :)

Sounds like a good alternative if people are looking for one. 
Cheers!
